proto/config/netcmn.proto

```diff
@@ -84,11 +84,21 @@ message ipspec {
   string gateway = 5;
   string domain = 6;
   string ntp = 7;
+  repeated string moreNtp = 10;
   repeated string dns = 8;
 
   // for IPAM management when dhcp is turned on.
   // If none provided, system will default pool.
   ipRange dhcpRange = 9;
+
+  DhcpOptionsOverride dhcp_options_override = 11;
+}
+
+// DhcpOptionsOverride is used to override or add dhcp options like f.e.
```

Comment:
Are the semantics to override or to ignore? Stated differently, if there are no ntp servers specified but ntp_server_exclusively is true, will the result be that no NTP server configured? Or will the ntp_server_exclusively be ignored if no NTP server is configured in the API?

Makes sense to be explicit and make the name match the intended semantics.

Reply:
I changed it to DhcpOptionsIgnore https://github.com/lf-edge/eve-api/pull/70/files#diff-35ea4ed74707f6ad6d17be169425a7acf4e25e09e9de81a0f7300ed80c5ec656R101

And I added a comment regarding what should happen if no ntp servers are specified: https://github.com/lf-edge/eve-api/pull/70/files#diff-35ea4ed74707f6ad6d17be169425a7acf4e25e09e9de81a0f7300ed80c5ec656R103